rest_framework_simplejwt/exceptions.py

```diff
@@ -26,7 +35,7 @@ def __init__(self, detail=None, code=None):
         if code is not None:
             detail_dict["code"] = code
 
-        super().__init__(detail_dict)
+        super().__init__(detail_dict)  # type: ignore
```

Comment:
Should this warning be ignored?

Currently, due to it being a Mixin, the parent class can't be deduced.